test/plots/barPlotTests.ts

```diff
@@ -1 +1,783 @@
 ///<reference path="../testReference.ts" />
+
+describe("Plots", () => {
+  describe("Bar Plot", () => {
+    describe ("orientations", () => {
+      it("rejects invalid orientations", () => {
+        assert.throws(() => new Plottable.Plots.Bar("diagonal"), Error);
```

Comment:
It's the only `assert()` in the test, so it'd be a bit redundant with the test title itself.